mitmproxy/net/http/http1/read.py

```diff
@@ -111,7 +111,8 @@ def _read_request_line(line: bytes) -> Tuple[str, int, bytes, bytes, bytes, byte
                 raise ValueError
         else:
             scheme, rest = target.split(b"://", maxsplit=1)
-            authority, path_ = rest.split(b"/", maxsplit=1)
+            authority, *paths_ = rest.split(b"/", maxsplit=1)
+            path_ = paths_[0] if paths_ else b""
```

Comment:
Can you use `authority, _, path = rest.partition(b"/")` here? That should make stuff a bit cleaner.

Reply:
Absolutely! Never thought of it, for some reason :)